sec/iterated.tex

```diff
@@ -49,31 +62,37 @@ \section{Iterated monotone path polytopes}
 Note also that in order for this definition to make sense, one needs to use crucially the functoriality of the fiber polytope construction \cite[Lemma 2.3]{BilleraSturmfels92}.
 \anibal{Why?}
```

Comment:
Otherwise the definition of the P_k does not make sense; when we write Sigma (Sigma(), Sigma()), we use the fact that the fiber polytope is sent to the other fiber polytope

Reply:
I realize that you know the answers to your questions, but just want me to clarify for the reader